terraform/deployments/github/README.md

```diff
@@ -33,6 +33,64 @@ protecting the `main` branch.
   ```shell
   GITHUB_TOKEN=<token> gds aws govuk-production-poweruser -- terraform apply
   ```
+## Adding existing repositories
+
+To manage an existing repository using the GOV.UK GitHub Infrastructure configuration, it needs to be imported into terraform state. Otherwise terraform apply will fail because it will attempt to create a repository that already exists. 
```

Comment:
```suggestion
To manage an existing repository using the GOV.UK GitHub Infrastructure configuration, it needs to be imported into terraform state. Otherwise terraform apply will fail attempting to create a repository that already exists. 
```